### PR TITLE
Assignment 1 Solution

### PR DESCRIPTION
**Pull request by mirjetash**

### CODE REVIEW

File: mirjetash/assignment-1/main.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Example usage of functions from word_anagram.py and sentence_anagram,py
```
Comment (ariava) [inline mirjetash/assignment-1/main.py]: This is cool, you can also try to use argparse (https://docs.python.org/3/library/argparse.html) which is a library that should let you handle command line options in an easier way.

File: mirjetash/assignment-1/sentence_anagram_test.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+from sentence_anagram import are_sentences_anagrams, are_sentences_by_words_anagrams
+
+
+class SentenceAnagramTest(unittest.TestCase):
+    """Test calss forfunctions in sentence_anagram. """
+
+    def test_sentence_anagrams(self):
+        sentence1 = "Is this one sentence?"
+        sentence2 = "This is one sentence!"
+        self.assertEqual(are_sentences_anagrams(sentence1, sentence2, case_sensitive=False), True)
```
Comment (ariava) [inline mirjetash/assignment-1/sentence_anagram_test.py]: You can use assertTrue and assertFalse (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue) which will make the code more readable.

File: mirjetash/assignment-1/word_anagram_test.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import unittest
+from word_anagram import are_words_anagrams 
+
+class WordAnagramTest(unittest.TestCase):
+    """Test class for functions in word_anagram. """
+
+    def test_word_anagrams(self):
+        word1 = "a"
```
Comment (ariava) [inline mirjetash/assignment-1/word_anagram_test.py]: Maybe also test against the empty string. The more corner cases you can come up with, the better! Testing looks great though.

File: mirjetash/assignment-1/main.py (hunk 1 above)
Comment (mirjetash) [reply mirjetash/assignment-1/main.py]: Thank you! I will check it in the future.

File: mirjetash/assignment-1/word_anagram_test.py (hunk 3 above)
Comment (ariava) [reply mirjetash/assignment-1/word_anagram_test.py]: Please address at least this comment. If you do the change and then commit them on the same branch, this pull request will be modified automatically,

Comment (mirjetash) [reply mirjetash/assignment-1/word_anagram_test.py]: I have just made the necessary changes.
